spec/ic.did

```diff
@@ -29,8 +28,9 @@ type ecdsa_curve = variant { secp256k1; };
 type satoshi = nat64;
 
 type bitcoin_network = variant {
-  mainnet;
-  testnet;
+  Mainnet;
+  Testnet;
+  Regtest;
```

Comment:
What does `Regtest` mean?

Reply:
FYI, https://developer.bitcoin.org/examples/testing.html#regtest-mode
For local bitcoin development, `regtest` is the chain being connected. The relevant [PR here](https://github.com/dfinity/sdk/commit/8f14d6688364d182d0b9b36497fc84bda3c9012c).

Reply:
Regtest is a tool that we're choosing to add to facilitate local development, but it's not part of production, so IMO it does not belong to the interface spec.